src/applications/vaos/referral-appointments/components/CCAppointmentCard.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
```

Comment:
I feel like this card already exists in VAOS. Maybe not though, I'll dig around

Reply:
It does exist, but does not quite match the styling that is in figma and it is built around the `appointment` object so it can't be used as a plug and play. So I took the styles from that and matched them to the UX designs

Reply:
ah gotcha, makes sense to me then.